packages/Webkul/Customer/src/Models/CustomerAddress.php

```diff
@@ -8,5 +9,18 @@ class CustomerAddress extends Model implements CustomerAddressContract
 {
     protected $table = 'customer_addresses';
 
-    protected $fillable = ['customer_id' ,'address1', 'country', 'state', 'city', 'postcode', 'phone', 'default_address'];
+    protected $fillable = [
+        'customer_id',
+        'company_name',
+        'vat_id',
+        'address1',
+        'address2',
+        'address3',
```

Comment:
Hi Herbert,
there is no field "address3" in DB schema. So remove this from fillable or add a migration to alter the table.

Reply:
Correct. I have removed the line.

Reply:
Nice.
I would also suggest to split the existing field "name" in table customer_addresses into "first_name" and "last_name" as it is in table customers.

Happy coding.